portals/publisher/source/src/app/components/Apis/Details/Endpoints/GeneralConfiguration/CertificateUsage.jsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fix License use correct year and check all your nee files .Configure your idea to add this automatically

Reply:
fixed